src/layout/Stepper/Step.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+export interface StepProps extends PropsWithChildren {
+  /**
+   * Optional Text of the marker
+   */
+  markerLabel?: string;
+
+  /**
+   * CSS Classname to applied to the step
+   */
+  className?: string;
+}
+
+export const Step: FC<StepProps> = ({ children, className }) => (
+  <div className={className}>{children}</div>
```

Comment:
This probably needs a role?

Reply:
Do you mean this one https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles?

Reply:
Probably `listitem` but I'm not sure